src/test/scala/com/fulcrumgenomics/commons/io/IoTest.scala

```diff
@@ -62,6 +75,15 @@ class IoTest extends UnitSpec {
     Io.assertReadable(Io.StdIn)
   }
 
+  it should "not throw an exception for Unix fifos" in { if (hasMkfifo) {
```

Comment:
Is there any way to do something like `(if (hasMakfifo) it else ignore) should ...` so that it's clear the test is ignored when `mkfifo` isn't present?

Reply:
I went with `assume(hasMkfifo)`